EmployeeSort/Employee.rb

```diff
@@ -0,0 +1,32 @@
+require "test/unit"
+
+class Employee
+include Comparable
+attr_accessor :name
+
+  def <=>nextEmployeeObject
+    self.name <=> nextEmployeeObject.name
+  end
+
+  def inspect
```

Comment:
sir the inspect was defined so that i could get a readable format of object and hence could test it.